datalad/crawler/nodes/tests/test_misc.py

```diff
@@ -62,6 +115,28 @@ def test_assign():
     eq_(list(gen(datadup)), [{'x': 'value', 'g': 'y z'}])
 
 
+def test__act_if():
+    values = {'x': 'y', 'a': 'b'}
+    data = {'x': 'y'}
+    datamatch = {'x': 'y', 'a': 'b'}
+
+    # matched = false
+    gen = _act_if(values, False, True)
```

Comment:
to not bother matching False, True to `re` and `negate` -- use them as keyword args here.  Also test with "default" values (i.e. your False, False below) - i.e. not providing any value here